packages/slice-machine/lib/env/semver.ts

```diff
@@ -65,12 +73,18 @@ export const createComparator =
         ? MessageByManager.YARN(manifest.name, onlinePackage.version)
         : MessageByManager.NPM(manifest.name, onlinePackage.version);
 
+      const versions = await getAvailableVersionInfo(
+        manifest.name,
+        manifest.version
+      );
+
       return {
         currentVersion: manifest.version,
         onlinePackage,
         updateAvailable,
```

Comment:
updateAvailable should come from the fact that there is nothing in the versions object.
```suggestion
        updateAvailable: availableUpdates.patch || availableUpdates.minor || availableUpdates.major
```